tests/golden/check-local-refs/dir1/dir2/d2f1.md

```diff
@@ -9,23 +9,32 @@
 [bad-cf-ref](#bad)
 
 # Relative links
-[existing-file-rel](d2f2.md)
-[existing-file-rel](./d2f2.md)
-[existing-file-rel](../dir2/.././d1f1.md)
+[existing-file-rel-1](d2f2.md)
+[existing-file-rel-2](./d2f2.md)
+[existing-file-rel-3](../dir2/.././d1f1.md)
+[existing-file-rel-4](d2f3.yaml)
 
+[existing-dir-rel-1](..)
+[existing-dir-rel-2](../dir2)
+[existing-dir-rel-3](../dir2/)
 
 [existing-anchor-rel-1](d2f2.md#existing-anchor-d2f2)
 [existing-anchor-rel-2](./d2f2.md#existing-anchor-d2f2)
 [existing-anchor-rel-3](../dir2/../d1f1.md#existing-anchor-d1f1)
 
 [bad-file-rel](../a/b/c/unexisting-file.md)
+[bad-slash-file-rel](d2f2.md/)
+[bad-casing-file-rel](D2F2.md/)
+[bad-casing-folder-rel](../DIR2)
 
 [bad-anchor-rel-1](d2f2.md#bad-anchor)
 [bad-anchor-rel-2](unexisting-file.md#bad-anchor)
 
 # Absolute links
 Should be correct when root is `/tests/golden/check-local-refs`:
 [file-abs-1](/dir1/./d1f1.md)
```

Comment:
Let's also add an absolute link + to a file + with a trailing slash

`[file-abs-2](/dir1/./d1f1.md/)`

Reply:
Done